src/components/dev-hub/survey-banner.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import Button from './button';
+import Icon from '@leafygreen-ui/icon';
+import { fontSize, screenSize, size, colorMap } from './theme';
+
+const Footer = styled('div')`
+    display: ${({ showSurvey }) => (showSurvey ? `block` : `none`)};
+    background-color: ${colorMap.greyDarkTwo};
+    position: fixed;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border: 1px solid ${colorMap.greyDarkOne};
+`;
+
+const FooterContent = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    @media ${screenSize.mediumAndUp} {
+        margin: 0 2% 0 15%;
+    } ;
+`;
+
+const SurveyContent = styled('div')`
+    @media ${screenSize.mediumAndUp} {
+        display: flex;
+    }
+`;
+
+const SurveyText = styled('div')`
+    font-family: 'Fira Mono', monospace;
+    font-size: ${fontSize.tiny};
+    padding: 18px;
+    @media ${screenSize.upToMedium} {
+        padding: ${size.xsmall} ${size.xsmall} 0px ${size.xsmall};
+    }
+`;
+
+const SurveyButtonContainer = styled('div')`
+    display: 'flex';
+    padding: ${size.xsmall} 0px ${size.xsmall} ${size.xsmall};
+`;
+
+const CloseButtonContainer = styled('div')`
+    padding: 18px;
+`;
+
+const StyledButton = styled(Button)`
+    font-size: ${fontSize.tiny};
+    padding: ${size.default} ${size.medium};
+`;
+
+const StyledIcon = styled(Icon)`
+    cursor: pointer;
+`;
+
+const surveyLink =
+    'https://mongodb.co1.qualtrics.com/jfe/form/SV_24x6XG9EVxE0dOm';
+
+const SurveyBanner = () => {
+    const [showSurveyBanner, setShowSurveyBanner] = useState(true);
```

Comment:
Do u mean gatsby-ssr here?

Reply:
Not necessarily there but we should default in this component here to hide the footer UNLESS local storage comes back empty to prevent that flash I sent over on Slack